docs/user-guide/concepts.md

```diff
@@ -218,6 +221,64 @@ probably not a great idea.
 You can create a subnamespace from the command line via `kubectl hns create child
 -n parent`.
 
+<a name="hrq">
+
+### Hierarchical resource quotas (HRQs)
+
```

Comment:
Can you please add a note here that HRQ is beta in HNC v1.1? Like this:

https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/concepts.md#managed-labels-and-annotations

Similarly in the other two docs?